examples/glyphs/maps_cities.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import print_function
+
+import os
+import json
+
+import numpy as np
+
+from bokeh.browserlib import view
+from bokeh.document import Document
+from bokeh.embed import file_html
+from bokeh.models.glyphs import Circle
+from bokeh.models import (
+    GMapPlot, Range1d, ColumnDataSource, LinearAxis,
+    PanTool, WheelZoomTool, BoxSelectTool,
+    BoxSelectionOverlay, GMapOptions,
+    NumeralTickFormatter, PrintfTickFormatter)
+from bokeh.resources import INLINE
+
+x_range = Range1d()
+y_range = Range1d()
+
+map_options = GMapOptions(lat=35, lng=-100, zoom=4)
+
+plot = GMapPlot(
+    x_range=x_range, y_range=y_range,
+    map_options=map_options,
+    title = "Austin"
+)
+plot.map_options.map_type="hybrid"
+
+city_data = json.load(open(os.path.expanduser('~/us_cities.json'), 'rt'))
```

Comment:
@mattpap can you write up a paragraph or two about uploading to S3 and send it to bokeh-internal? You have done this most recently and more than anyone else.